charts/csm-authorization/values.yaml

```diff
@@ -67,6 +68,18 @@ authorization:
     annotations: {}
       # nginx.ingress.kubernetes.io/backend-protocol: "GRPC"
 
+  # storage-service ingress configuration
+  storageServiceIngress:
+    ingressClassName: nginx
+
+    # additional host rules for the tenant-service ingress
```

Comment:
this comment should say storage service ingress. Can you also update the comment for the roleServiceIngress host parameter? That says tenant-service too